aws/validators.go

```diff
@@ -2446,6 +2446,29 @@ func MapKeysDoNotMatch(r *regexp.Regexp, message string) schema.SchemaValidateFu
 	}
 }
 
+func MapKeyInSlice(valid []string, ignoreCase bool) schema.SchemaValidateFunc {
```

Comment:
this may want to be moved to upstream sdk

Reply:
See also https://github.com/hashicorp/terraform-plugin-sdk/issues/649.